test/e2e/storage/persistent_volumes.go

```diff
@@ -18,6 +18,7 @@ package storage
 
 import (
 	"fmt"
+	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
```

Comment:
`e2elog` will be removed later.
Please use `"k8s.io/kubernetes/test/e2e/framework"` instead.
`framework.Logf...`